packages/vats/src/lib-board.js

```diff
@@ -199,6 +199,11 @@ const makeSlotToVal = state => {
   return slotToVal;
 };
 
+/** @type {import('@endo/marshal').MakeMarshalOptions} */
+const useSmallCaps = harden({
+  serializeBodyFormat: 'smallcaps',
+});
+
```

Comment:
I prefer that you not factor this out, but rather repeat the object literal inline in both call sites. The reason is that this is an options bag, and each call site could in theory want to add their own options over time. Think of the options like `serializeBodyFormat` as approximating name-based parameters as opposed to positional parameters.